src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -96,6 +96,7 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
                       help='Update Domain count. If unspecified, server picks the most optimal number like 5. For the latest defaults see https://docs.microsoft.com/en-us/rest/api/compute/availabilitysets/availabilitysets-create')
 register_cli_argument('vm availability-set create', 'platform_fault_domain_count', type=int, help='Fault Domain count.')
 register_cli_argument('vm availability-set create', 'validate', help='Generate and validate the ARM template without creating any resources.', action='store_true')
+register_cli_argument('vm availability-set convert', 'platform_fault_domain_count', type=int, help="New Fault Domain count if provided. Use 'az vm list-skus -l <location>' to check out available count")
```

Comment:
You should probably use the "populator-command" feature in `help.py` to take advantage of the standard formatting here.